nestkernel/synaptic_element.cpp

```diff
@@ -47,7 +47,7 @@ nest::SynapticElement::SynapticElement()
   , z_connected_( 0 )
   , continuous_( true )
   , growth_rate_( 1.0 )
-  , tau_vacant_( 10.0 )
+  , tau_vacant_( 0.1 )
```

Comment:
@sdiazpier I am a bit surprised by the change of the default value of `tau_vacant_` from 10.0 to 0.1. This probably has no consequence, since the value was not used anywhere in the past, but could you just confirm that the change is intentional?